templates/SSO/aws-sso.yaml

```diff
@@ -58,6 +64,13 @@ Resources:
       Description: !Sub '${permissionSetName} access to AWS organization'
       InstanceArn: !Ref instanceArn
       ManagedPolicies: !If [ includeManagedPolicies, !Ref managedPolicies, !Ref 'AWS::NoValue' ]
+      CustomerManagedPolicyReferences:
+        !If
+          - includeCustomerManagedPolicy
+          - # If we are setting a value, we are also embedding it as a single item in a list
```

Comment:
this, dash with comment, looks wrong.  value probably should look something like value for ManagedPolicies, no?

Reply:
We have two lists here: the outer list used by the `!If` where the second element in the list is our desired output, and the output value itself, which is a list with a single element. I've confirmed with a manual sceptre deploy that this works as expected.